packages/components/molecules/f-rating/README.md

```diff
@@ -69,11 +69,13 @@ There may be props that allow you to customise its functionality.
 
 The props that can be defined are as follows (if any):
 
-| Prop          |   Type   | Required | Default | Description                                                                                                            |
-| :---          |:--------:|:--------:|:-------:|:-----------------------------------------------------------------------------------------------------------------------|
-| `starRating`  | `Number` |   Yes    |    -    | Sets the displayed rating (filled stars). i.e. for `2 out of x`, 2 would be the `starRating`                           |
-| `maxStarRating`  | `Number` |    No    |    5    | Sets the maximum number of stars that the rating is set against. i.e. for `x out of 5`, 5 would be the `maxStarRating` |
-| `starRatingSize`  | `String` |    No    | 'small' | Sets the component size. By default the component will use `small` the other options are `medium` & `large`            |
+| Prop          |   Type   | Required | Default | Description                                                                                                            | Options |
+| :---          |:--------:|:--------:|:-------:|:-----------------------------------------------------------------------------------------------------------------------|---------|
+| `starRating`  | `Number` |   Yes    |    -    | Sets the displayed rating (filled stars). i.e. for `2 out of x`, 2 would be the `starRating`                           | -
+| `maxStarRating`  | `Number` |    No    |    5    | Sets the maximum number of stars that the rating is set against. i.e. for `x out of 5`, 5 would be the `maxStarRating` | -
+| `starRatingSize`  | `String` |    No    | 'small' | Sets the component size. By default the component will use `small`.                                                    | `small` `medium` `large`
+| `ratingDisplayType`  | `String` |    No    |  null   | Sets the descriptive text which will be displayed next to the component. By default there will be no text.             | `noRating` `short` `medium` `long`
+| `totalReviews`  | `Number` |    No    |  null   | Sets the total number of reviews to display. Example: `View 499 reviews` where `499` is the total.                     | -
```

Comment:
Would `reviewCount` be a bit more succinct?

Reply:
Not great with naming, ill change it 👍

Reply:
Fixed.